cssom-1/Overview.bs

```diff
@@ -949,6 +953,25 @@ The <dfn method for=CSSStyleSheet>deleteRule(<var>index</var>)</dfn> method must
  <li><a>Remove a CSS rule</a> in the <a for=CSSStyleSheet>CSS rules</a> at <var>index</var>.
 </ol>
 
+Legacy CSSStyleSheet members
+----------------------------
+
+The <dfn attribute for=CSSStyleSheet>rules</dfn> attribute must follow the same steps as {{CSSStyleSheet/cssRules}}, and return the same object {{CSSStyleSheet/cssRules}} would return.
+
+The <dfn attribute for=CSSStyleSheet>removeRule(<var>index</var>)</dfn> method must run the same steps as {{CSSStyleSheet/deleteRule()}}.
+
+The <dfn method for=CSSStyleSheet>addRule(<var>selector</var>, <var>block</var>, <var>optionalIndex</var>)</dfn> method must run the following steps:
+
+<ol>
+ <li>Let <var>rule</var> be an empty string.
+ <li>Append <var>selector</var> to <var>rule</var>.
+ <li>Append <code>" { "</code> to <var>rule</var>.
+ <li>If <var>block</var> is not empty, append <var>block</var>, followed by an space, to <var>rule</var>.
+ <li>Append <code>"}"</code> to <var>rule</var>
+ <li>Let <var>index</var> be <var>optionalIndex</var> if provided, or the number of <a for=CSSStyleSheet>CSS rules</a> in the stylesheet otherwise.
+ <li>Call {{CSSStyleSheet/insertRule()}}, with <var>rule</var> and <var>index</var> as arguments.
```

Comment:
Should return its return value, right?

Reply:
No, it returns -1:

http://webkit.crisal.io/webkit/rev/6cc00baaf9835dc4abdf7b16e72cff60ecd7bb88/Source/WebCore/css/CSSStyleSheet.cpp#324

Fixed.